addons/zylann.hterrain/hterrain_detail_layer.gd

```diff
@@ -203,6 +203,14 @@ func get_custom_shader():
 	return custom_shader
 
 
+func set_density(v):
+	density = clamp(v, 0.0, 1.0)
```

Comment:
This won't update existing grass in the editor, but that should be ok for now

Reply:
Oh, you have to pan away! I wonder if hiding and recycling  the cells would work... 🤔

Reply:
Regenerating the multimesh and iterate through current chunks could be enough